src/webgpu/api/operation/rendering/blending.spec.ts

```diff
@@ -20,7 +20,71 @@ import { float32ToFloat16Bits } from '../../../util/conversion.js';
 import { TexelView } from '../../../util/texture/texel_view.js';
 import { textureContentIsOKByT2B } from '../../../util/texture/texture_ok.js';
 
-export const g = makeTestGroup(GPUTest);
+class BlendingTest extends GPUTest {
+  createRenderPipelineForTest(
+    format: GPUTextureFormat,
+    blendComponent: GPUBlendComponent | undefined
+  ): GPURenderPipeline {
+    return this.device.createRenderPipeline({
+      layout: 'auto',
+      fragment: {
+        targets: [
+          {
+            format,
+            blend: {
+              color: blendComponent ?? {},
+              alpha: blendComponent ?? {},
```

Comment:
This assumes blendComponent is the same for color and alpha, but we'll probably need tests where they're not the same. I think we should change this method to take a whole `GPUColorTargetState` instead of the `format` and `blendComponent` separately.

Reply:
ok, I change the parameter to `GPUColorTargetState`.